Index: subversion/bindings/swig/ruby/svn/util.rb
===================================================================
--- subversion/bindings/swig/ruby/svn/util.rb	(revision 1221556)
+++ subversion/bindings/swig/ruby/svn/util.rb	(working copy)
@@ -36,7 +36,7 @@ module Svn
   module Util #:nodoc:
     module_function
     def to_ruby_class_name(name)
-      name.split("_").collect do |x|
+      name.to_s.split("_").collect do |x|
         "#{x[0,1].upcase}#{x[1..-1].downcase}"
       end.join("")
     end
Index: subversion/bindings/swig/ruby/svn/info.rb
===================================================================
--- subversion/bindings/swig/ruby/svn/info.rb	(revision 1221556)
+++ subversion/bindings/swig/ruby/svn/info.rb	(working copy)
@@ -229,7 +229,9 @@ module Svn
 
     def parse_diff_unified(entry)
       in_content = false
-      entry.body.each do |line|
+      # accomodation for ruby 1.9 and 1.8
+      each_meth = entry.body.respond_to?(:each_line) ? :each_line : :each
+      entry.body.send(each_meth) do |line|
         case line
         when /^@@/
           in_content = true

